.github/workflows/run-playwright-browserstack.yml

```diff
@@ -0,0 +1,56 @@
+name: Run Playwright on Browserstack
+
+on:
+  workflow_dispatch:
+    inputs:
+      user_name:
+        description: browserstack user name
+        required: true
+        type: string
+      access_key:
+        description: browserstack access key
+        required: true
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    # define node version
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # setting up node
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      # install
+      - name: Install Dependencies
+        if: steps.cache-nodemodules.outputs.cache-hit != 'true'
+        run: |
+          npm ci
+
+      # install playwright
+      - name: Install Playwright with Chrome
+        run: |
+          npx playwright install --with-deps
+
+      # build
+      - name: Build
+        run: |
+          npm run build:prod
+
+      # Run E2E Tests
+      - name: PlayWright Test
+        run: |
+          npm run e2e:playwright:ci-browserstack
+        env:
+          BROWSERSTACK_USER_NAME: ${{ inputs.user_name }}
+          BROWSERSTACK_ACCESS_KEY: ${{ inputs.access_key }}
```

Comment:
You can't pass the credentials this way, they are visible in the workflow logs for everyone!
Please lookup the Github solution for how to deal with credentials and secrets.
Old workflow logs needs to be deleted therefore.
Please also re-check the logs after fixing that there are no credentials logged anywhere.

Reply:
Done, the credentials are now masked in the log